### PR TITLE
add event "onload"

### PR DESCRIPTION
**Pull request by kyob1010**
I fork the repo, and put the code in "scriptserver-onload" to it

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -42,4 +42,16 @@ module.exports = function() {
...
+    const result = log.match(/]:\sDone/);
+    // server launch done
+    if(result) {
+      // send load event
+      server.emit('load');
+      // end listen
+      server.removeListener('console', load_listener);
+    }
+  }
+  server.on('console', load_listener);
```
Comment (garrettjoecox) [inline index.js]: Could we make this more syntactically consistent with the rest of the listeners? 

Does `load` feel a little vague? maybe `start`? I'm not sure..

Also add a timestamp to the emit

Comment (kyob1010) [inline index.js]: That's a great idea!
I'll rewrite it later

Comment (kyob1010) [inline index.js]: Ok. I'm already done.

**Comment by kyob1010** [comment]
I found my mistake on event "onstart", and it affect event "onexit".
It works fine now.

b.t.w The code in event "onexit" seems not beauty enough, but I thought it is the only solution now.

**Comment by garrettjoecox** [comment]
Looks great

**Pull request merged**